lib/tessel/commands.js

```diff
@@ -5,7 +5,7 @@ module.exports.scanWiFi = function() {
   return ['ubus', 'call', 'iwinfo', 'scan', '{"device":"wlan0"}'];
 };
 module.exports.stopRunningScript = function() {
-  return ['/etc/init.d/tessel-app', 'stop'];
+  return ['killall', 'node'];
```

Comment:
Yeah, but I think that can be deferred; right now it's incorrect behavior and we have no other node scripts running afaik.